src/main/scala/chisel3/iotesters/ChiselSpec.scala

```diff
@@ -10,14 +10,14 @@ import chisel3.testers._
 
 /** Common utility functions for Chisel unit tests. */
 trait ChiselRunners extends Assertions {
+  val backends = Seq("firrtl", "verilator", "vcs")
   def runTester(t: => BasicTester, additionalVResources: Seq[String] = Seq()): Boolean = {
     TesterDriver.execute(() => t, additionalVResources)
   }
   def assertTesterPasses(t: => BasicTester, additionalVResources: Seq[String] = Seq()): Unit = {
     assert(runTester(t, additionalVResources))
   }
   def elaborate(t: => Module): Unit = Driver.elaborate(() => t)
```

Comment:
This fails with:

    .../src/main/scala/chisel3/iotesters/ChiselSpec.scala:20: value elaborate is not a member of object chisel3.iotesters.Driver
    [error]   def elaborate(t: => Module): Unit = Driver.elaborate(() => t)
    [error]                                              ^
    [error] one error found
    [error] (compile:compileIncremental) Compilation failed